system/modules/isotope/library/Isotope/Model/Address.php

```diff
@@ -203,7 +203,21 @@ public function getTokens($arrFields = null)
 
                 list($country, $subdivion) = explode('-', $this->subdivision);
 
-                $arrTokens['subdivision']      = $arrSubdivisions[strtolower($country)][$this->subdivision];
+                foreach ($arrSubdivisions[strtolower($country)] as $v) {
+                    // Support multi-dimensional arrays (e.g. subdivisions of United Kingdom)
+                    if (is_array($v)) {
+                        foreach ($v as $vv) {
+                            if ($vv[$this->subdivision]) {
+                                $arrTokens['subdivision'] = $vv[$this->subdivision];
+                                break;
+                            }
+                        }
+                    } else {
+                        $arrTokens['subdivision'] = $arrSubdivisions[strtolower($country)][$this->subdivision];
```

Comment:
I have tested it and it works as expected. We are using the foreach-value in the loop, see ```$v``` on line 209. If that is not an array on the other hand, then the loop is immediately broke in 217.